### ISSUE TITLE
Generating wrong image url with domain name

### ISSUE THREAD

**Issue by OoBook** [area17/twill#1591]
## Description

After updated from 2.5.2 version to 2.7, the image url started to form incorrectly. urlGenerator adds domain_name at the head of url string. 

### Steps to reproduce
I've uploaded an image, then rendered front page. Similarly, this error occurs at image uploader modal.

### Expected result
Media Library should show the demanded image accurately.

### Actual result
The image url is getting generated in wrong way.

### Versions

Twill:
old version: 2.5.2
current version: 2.7

Laravel: 8.0

Php: 7.4

DB Engine: InnoDB 10.6.5

**Comment by ifox** [comment]
Hi @OoBook, can you please share which image service you're using as well as your APP_URL?

**Comment by OoBook** [comment]
Thanks for answer. Image service that we're using is "glide", we've set APP_URL as 'default-cms.test' on local. With this configuration, after I upload the image, image source is following: 
```
src="/default-cms.test/img/{image_id}/{image_name}.{extension}"
```

As before said, there is no such error in the version '2.5.2'

**Comment by OoBook** [comment]
On twill\config\glide.php file, you've changed 

```
-- 'base_url' => env('GLIDE_BASE_URL', null),

++ 'base_url' => env('GLIDE_BASE_URL', config('app.url')),

```

Problem results from this change.

**Comment by ifox** [comment]
Indeed, thanks @OoBook! This is actually already fixed in 2.8.0-rc.1. if you're willing to, please give rc.2 a try. We're about to release the stable.

**Comment by OoBook** [comment]
Thanks @ifox! I'll look through it.
